### ISSUE TITLE
data.R has some IPHC functions - check they can be deleted since are in gfiphc

### ISSUE THREAD

**Issue by andrew-edwards** [pbs-assess/gfsynopsis#139]
These should be deleted if they're doing what is now done in gfiphc (which is the version being actively developed). Functions here are probably just old. Do this for next gfsynopsis report.

**Comment by seananderson** [comment]
Yep. Sometime this year I hope—after all the 2020 survey data are in the databases. There's also a bunch of redundant data functions in gfplot that are now in gfdata to be cleaned up.

**Comment by andrew-edwards** [comment]
The two IPHC-related functions in data.R both call:
```
gfiphc::cache_pbs_data_iphc
```
so need to stay (think they may have earlier been some other ones that may have been the intent of this issue).

 I would close, but let Sean deal with his previous comment if necessary..
